src/System.Collections.NonGeneric/ref/System.Collections.NonGeneric.csproj

```diff
@@ -1,6 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Configurations>netcoreapp-Debug;netcoreapp-Release;uap-Debug;uap-Release</Configurations>
+    <DefineConstants>$(DefineConstants);INTERNAL_NULLABLE_ATTRIBUTES</DefineConstants>
```

Comment:
where is this used?

Reply:
(I can see attributes being used for implementation assembly but not the ref)

Reply:
for ref assembly

Reply:
> (I can see attributes being included for implementation assembly but not the ref)
oh I see what you mean, will remove